TradeItIosTicketSDK2Tests/TradeItLinkedBrokerManagerSpec.swift

```diff
@@ -596,10 +595,10 @@ class TradeItLinkedBrokerManagerSpec: QuickSpec {
                 expect(linkedBroker1.calls.forMethod("refreshAccountBalances(onFinished:)").count).to(equal(1))
                 expect(linkedBroker2.calls.forMethod("refreshAccountBalances(onFinished:)").count).to(equal(1))
             }
-
-            it("doesn't refresh the unauthenticated linkedBroker") {
-                expect(linkedBroker3.calls.forMethod("refreshAccountBalances(onFinished:)").count).to(equal(0))
-            }
+// Do we still want this behavior ?
```

Comment:
Before we were expecting to not refreshBalances on linkedBrokers that weren't authenticated. What behavior do we want ?